lib/foreman_scap_client/client.rb

```diff
@@ -36,7 +36,17 @@ def scan_command
                                else
                                  ''
                                end
-      "oscap xccdf eval #{fetch_remote_resources} #{profile} #{tailoring_subcommand} --results-arf #{results_path} #{config[@policy_id][:content_path]}"
+      "oscap xccdf eval #{fetch_remote_resources} #{local_files_subcommand} #{profile} #{tailoring_subcommand} --results-arf #{results_path} #{config[@policy_id][:content_path]}"
+    end
+
+    def local_files_subcommand
+      supports_local_file_option ? '--local-files /root' : ''
+    end
+
+    def supports_local_file_option
```

Comment:
In Ruby it's a common practice to include `?` in a method if it returns a boolean, so this is valid.
```suggestion
    def supports_local_file_option?
```